lighthouse-extension/app/src/popup.js

```diff
@@ -134,10 +134,15 @@ document.addEventListener('DOMContentLoaded', _ => {
     })
     .catch(err => {
       let {message} = err;
```

Comment:
can you change this line while you're there? Looks like it can be just `let message = err;`, no?